pkg/cloud/api/path.go

```diff
@@ -92,6 +123,23 @@ func (p Path) HasPrefix(prefix Path) bool {
 	return true
 }
 
+func isAllElemPrefix(path string) bool {
+	if len(path) != 2 {
+		return false
+	}
+	if path[0] != pathSliceIndex {
+		return false
+	}
+	return path[1] == arrayAllElem
+}
+
+func isArrayIndex(path string) bool {
```

Comment:
We should always keep the same naming convention as the constant to make it easier to understand.

change to `func isPathSliceIndex(element string)`

The string argument is not a "Path" type.